src/view/View.js

```diff
@@ -1076,18 +1076,9 @@ new function() { // Injection scope for event handling on the browser
         mousemove = 'pointermove MSPointerMove';
         mouseup = 'pointerup pointercancel MSPointerUp MSPointerCancel';
     } else {
```

Comment:
Unfortunately there seems to be some possible breaking behaviour, which I'm not fully aware of atm https://github.com/paperjs/paper.js/pull/1810#issuecomment-633153793

Reply:
Using `PointerEvent` does fix it for me but I still see no harm in always listening to all the events instead of depending on the userAgent string. The mouse events shouldn't be called after calling `preventDefault` on the touch events anyways.